### PR TITLE
Scheme https

### PR DESCRIPTION
**Pull request by rranauro**
The 0.0.5 upgrade to AssayDepot implemented a new request library. However, it was not tested on `https://` protocol servers. This Pull Request fixes that.

Additional tests were added to check for secure QuotedWare responses. Negative from Storefront URLs, positive from Backoffice URLs.

Some improvements to the README.md to bring it closer in line with the current API documentation and Scientist.com brand.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,32 @@ Or install it yourself as:
...
 ## Basic Usage
 
+### Storefront
+
 ```ruby
 require 'assaydepot'
 AssayDepot.configure do |config|
   config.access_token = "1234567890"
-  config.url = "https://www.assaydepot.com/api"
+  config.url = "https://app,scientist.com"
```
Comment (cpetersen) [inline README.md:31]: Little type (a `,` that should be a `.`)

File: README.md [hunk 2]
```diff
@@ -22,16 +22,32 @@ Or install it yourself as:
...
 wares.total
 ```
 
+### Backoffice
+
+```ruby
+require 'assaydepot'
+AssayDepot.configure do |config|
+  config.access_token = "1234567890"
+  config.url = "https://backoffice,scientist.com"
```
Comment (cpetersen) [inline README.md:43]: Same `, => .`

File: spec/assaydepot_spec.rb [hunk 3]
```diff
@@ -117,7 +116,7 @@
         end
 
         it "should have a description" do
-          provider_result["provider"]["description"].should_not be_nil
+          provider_result["provider"]["keywords"].should_not be_nil
```
Comment (cpetersen) [inline spec/assaydepot_spec.rb:119]: Totally cool, but we should probably change the spec to be `should have keywords`

File: spec/assaydepot_spec.rb [hunk 4]
```diff
@@ -5,10 +5,9 @@
 describe AssayDepot do
   context "when accessing the api via token client credentials" do
     before(:all) do
-      site = "http://dev.scientist.com:3000"
       AssayDepot.configure do |config|
         config.access_token = ENV['ACCESS_TOKEN']
-        config.url = "#{site}/api/v2"
+        config.url = "#{ENV['SITE']}/api/v2"
```
Comment (cpetersen) [inline spec/assaydepot_spec.rb:10]: 👏

File: spec/assaydepot_backoffice_spec.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
+require 'assaydepot'
+require 'dotenv'
+Dotenv.load
+
+puts "Yes!"
```
Comment (cpetersen) [inline spec/assaydepot_backoffice_spec.rb:5]: Awesome that you added more specs! This looks like an errant `puts` though.

**Review by cpetersen — CHANGES_REQUESTED** [review]
I left a couple of extremely minor comments! Sorry I didn't see this for so long! I guess without the slack notification, I just don't see pull requests...

Along those lines, I think I made a bunch of changes off of an old version of the API. I could use your help getting them merged (or maybe rewritten). Maybe we could pair up?

This looks great

**Comment by rranauro** [comment]
> I left a couple of extremely minor comments! Sorry I didn't see this for so long! I guess without the slack notification, I just don't see pull requests...
> 
> Along those lines, I think I made a bunch of changes off of an old version of the API. I could use your help getting them merged (or maybe rewritten). Maybe we could pair up?
> 
> This looks great

Well...you were having a baby and I found a work-around for Sci2lead so I did not think this one was so critical for your immediate attention. Thanks for the comments!

**Comment by rranauro** [comment]
> I left a couple of extremely minor comments! Sorry I didn't see this for so long! I guess without the slack notification, I just don't see pull requests...
> 
> Along those lines, I think I made a bunch of changes off of an old version of the API. I could use your help getting them merged (or maybe rewritten). Maybe we could pair up?
> 
> This looks great

I am happy to take a stab at getting the specs running for [PR#4](https://github.com/assaydepot/assaydepot-rb/pull/4) let me see how far I get.

File: spec/assaydepot_spec.rb (hunk 3 above)
Comment (rranauro) [reply spec/assaydepot_spec.rb]: I used this:

```
        it "should have a description" do
          expect(provider_result['provider']).to have_key('keywords')
        end
```
If this is not the right right `matcher` please help!

Comment (cpetersen) [reply spec/assaydepot_spec.rb]: Your code is great, I was referring to the line above where it says `it "should have a description" do`, I'll see if I can setup a suggested change. It's purely cosmetic.

File: spec/assaydepot_spec.rb [hunk 6]
```diff
@@ -117,7 +116,7 @@
         end
 
         it "should have a description" do
```
Comment (cpetersen) [inline spec/assaydepot_spec.rb:118]:
```suggestion
        it "should have keywords" do
```

**Review by cpetersen — COMMENTED** [review]
Last one!

File: spec/assaydepot_spec.rb (hunk 3 above)
Comment (rranauro) [reply spec/assaydepot_spec.rb]: your right! good catch.

**Review by rranauro — COMMENTED** [review]
Should be ready now.

**Review by cpetersen — APPROVED** [review]
:tada:

**Pull request merged**